src/idsk/components/intro-block/intro-block.scss

```diff
@@ -93,10 +93,10 @@
         color: govuk-colour("dark-grey");
     }
 
-    .idsk-intro-block__side-menu{
+    .idsk-intro-block__side-menu {
         padding: govuk-spacing(4);
-        @include mq($from: desktop){
-            margin-left: govuk-spacing(8);
+        @include mq($from: desktop) {
+            margin-left: 35px;
```

Comment:
Tu by uz nefungoval govuk-spacing(7)?

Reply:
To by už bolo 40px
![image](https://user-images.githubusercontent.com/31508588/142829965-392b7db8-d6d6-460a-aa47-bdd5048bf92c.png)